src/pheval/analyse/analysis.py

```diff
@@ -1,4 +1,3 @@
-# #!/usr/bin/python
```

Comment:
Why was this here in the first place?

Reply:
Definitely some type of mistake on my end, very likely to have happened at the beginning of the project